### ISSUE TITLE
v2.0.0beta.5 last horizontal button shows gradient

### ISSUE THREAD

**Issue by pgale** [Clooos/Bubble-Card#488]
I'm sure I raised this before but I can't find it now. Maybe it was on the HA forum post?

In a horizontal button stack, if all buttons fit in the horizontal space and there's no option to scroll them left/right at all, the right hand button still gets a gradient applied, implying that there is another button or two to see. This also just doesn't look nice. Can you remove the gradient or have an option for it? The existing 'hide gradient' option doesn't make a difference but I think that's for something else?

This is an image of what happens (on all browsers I have - ios, PC, Android

![Screenshot 2024-04-26 150608](https://github.com/Clooos/Bubble-Card/assets/513044/5d1027e8-fb6b-4883-9bb9-b1078b4e36ec)

You can see the 'Weather' button has a transparent gradient applied.

**Comment by Clooos** [comment]
Hi, can you tell me if this issue still occurs in the latest beta?

**Comment by pgale** [comment]
Thanks for the work on this :) will test it tomorrow when I'm home.

**Comment by Clooos** [comment]
Any news on that issue?

**Comment by pgale** [comment]
Hi, I got around the slight display issue of this by not using names in the buttons. However, I just tested again and the RHS gradient still appears when the buttons are just under the total width of the screen but not enough to trigger being able to swipe them left/right at all. So if you are aiming to have buttons that look fixed and not swipeable by carefully labelling them, it makes it look like the horizontal button stack has more to the right when it doesn't - or at the least, just doesn't look right.

My preference would be to have an option to turn the gradients off - or maybe just have a CSS modifier to make the gradients totally opaque (in fact, is that possible now by using the styling options or card_mod?)

I also want this for the top of the popup card itself as I move an element (time display) close to the top of the popup card as there's too much white space up there, particularly when trying to cram a lot into a single page popup. I have also requested an option to turn off the close button I think?
